### PR TITLE
ci: bump k8s versions used

### PR DESCRIPTION
**Pull request by JamieMagee**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -66,12 +67,13 @@ jobs:
     strategy:
       matrix:
         k8s:
+          # from https://hub.docker.com/r/kindest/node/tags
           - v1.14.10
-          - v1.15.11
-          - v1.16.9
-          - v1.17.5
+          - v1.15.12
```
Comment (viceice) [inline .github/workflows/ci.yaml:72]: @JamieMagee should we add a regex manager to auto update patch versions?

**Review by viceice — COMMENTED** [review]
otherwise LGTM

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (JamieMagee) [reply .github/workflows/ci.yaml:72]: What's the best way do to it, considering we have multiple versions? IMO it's not super important that these are up-to-date.

Comment (viceice) [reply .github/workflows/ci.yaml:72]: ok, we can leave it as is.

My idea would be to add a comment to each ling and use a regexmanager to match each line and disable major and minor updates for these files and that dep.

**Review by viceice — APPROVED** [review]

**Pull request merged**